.gitignore

```diff
@@ -62,3 +62,5 @@ sessions.db
 loadtestconfig.json
 
 /terraformenv
+.idea
+mattermost-load-test.iml
```

Comment:
nit: missing trailing newline? .editorconfig not yet setup?

Reply:
I'll add .editorconfig and fix